### PR TITLE
GridHelper: Fix clone()

### LINKED ISSUE #8451: feature: expose GridHelper arguments
like this.getSize = function () { return side; } etc

so we do not have to store them on app side?

### PR DESCRIPTION
**Pull request by Mugen87**
Fixes #8451, #8452 

The idea is to store the given parameters similar to the geometry generators and then use them when cloning the helper.

### CODE REVIEW

**Comment by mrdoob** [comment]
Nice!

File: src/helpers/GridHelper.js [hunk 1]
```diff
@@ -46,7 +53,31 @@ function GridHelper( size, divisions, color1, color2 ) {
...
-GridHelper.prototype.constructor = GridHelper;
+GridHelper.prototype = Object.assign( Object.create( LineSegments.prototype ), {
+
+	constructor: GridHelper,
+
+	copy( source ) {
+
+		LineSegments.prototype.copy.call( this, source );
+
+		this.parameters.size = source.parameters.size;
```
Comment (mrdoob) [inline src/helpers/GridHelper.js]: How about:

```js
Object.assign( this.parameters, source.parameters ); 
```

Comment (Mugen87) [reply src/helpers/GridHelper.js]: Seems to work like a charm^^

**Comment by mrdoob** [comment]
Thanks!

**Comment by Mugen87** [comment]
Other helpers can now implement their `copy()`/`clone()` methods in a similar fashion.

**Pull request merged**